frontend/src/main.ts

```diff
@@ -5,28 +5,39 @@ import VueMaterial from 'vue-material';
 
 import Vue from 'vue';
 import App from './App.vue';
-import router from './router';
+import router, { isErrorPage, goToMainPage, goToUnlockMetamaskPage, goToUnsupportedNetworkPage, goToInstallMetamaskPage, goToErrorPage } from './router';
 import API from './api';
 
 import { VueConstructor } from 'vue/types/vue';
 import registerPurpleTheme from './theme';
 
 Vue.use(VueMaterial);
-Vue.use(VueMaterial.MdIcon);
-
-window.addEventListener('load', async () => {
-  API.init();
-});
 
 registerPurpleTheme(Vue);
 
 Vue.config.productionTip = false;
 
 
 /* eslint-disable no-new */
-new Vue({
+var _Vue = new Vue({
```

Comment:
`const` instead of `var`?

Reply:
Also `vue` instad of `_Vue`. Let's try to avoid leading/trailing underscores in variable names. Unless this is a convention that I'm not aware of.